cypress/integration/team/create_a_team_spec.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Teams Suite', () => {
+    before(() => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+    });
+
+    it('Create a team', () => {
+
+        //  Click hamburger main menu
+        cy.get('#sidebarHeaderDropdownButton').click();
+
+        // dropdown menu should be visible then click create team
+        cy.get('#sidebarDropdownMenu').should('be.visible').within(function($el) {
+           cy.get('#createTeam').should('be.visible').click();
+        });
+
+        // input team name as Team Test
+        cy.get("input[type=text]").should('be.visible').type("Team Test");
+
+        // click next
+        cy.get("button[type=submit]").should('be.visible').click();
+
+        // input team url as team-test-e2e
+        cy.get("input[type=text]").should('be.visible').type("team-test-e2e");
```

Comment:
With regards to entering team URL of "team-test-e2e", this will fail on succeeding local dev test since team URL already exists at that period.  I'm thinking if it make sense to add random suffix to team URL. It can be something like:
```javascript
const teamURL = `team-test-${getRandomInt(9999).toString()}`;
cy.get('#teamURLInput').should('be.visible').type(teamURL);
```

`getRandomInt` is a new utility function where it returns random number when passed with max value (e.g. 9999).  Such can be located at `/cypress/utils/index.js`